docs/xsoftmx.md

```diff
@@ -50,7 +50,7 @@ When the heap shrinks, the garbage collector might release memory. The ability o
 
 :fontawesome-solid-pencil-alt:{: .note aria-hidden="true"} **Notes:**
 
-- When using `-Xgcpolicy:gencon`, `-Xsoftmx` applies only to the tenure area of the heap. In some cases the heap grows to greater than the `-Xsoftmx` value because the nursery portion grows, making the heap size exceed the limit that is set. See [`-Xmn`](xmn.md) for limiting the nursery size.
+- When using `-Xgcpolicy:gencon` with `-Xsoftmx`, then the proportion of heap space used for nursery within the `-Xsoftmx` limit, will be proportional to the maximum amount of nursery space specified (see [Xmn/Xmnx](xmn.md)) relative to the `-Xmx` value. For example, if the following was specified on the command line `-Xsoftmx2g -Xmnx4g  -Xmx8g`, nursery space will be allowed to use 1G/50% (4G/8G) of the specified `-Xsoftmx` value.
```

Comment:
division operator in 1G/50%  is confusing. it's either multiply operator (50% x 1G) or use 'of' (50% (4G/8G) of 1G...)

Reply:
Agreed, I'll fix that up

Reply:
Switched the passage to read:

"... nursery space will be allowed to use 50%(4G/8G) of the specified `-Xsoftmx` value, which in this example, would be 1G"

This reads better, and explains the proportion a bit better. Let me know if this can be further clarified